basiclti/web-ifp/src/bundle/vm/pick.vm

```diff
@@ -0,0 +1,57 @@
+<script type="text/javascript">
+	focus_path = [ "title" ];
+</script>
+
+<div class="portletBody">
+<script type="text/javascript" src="/library/js/headscripts.js"></script>
```

Comment:
When possible try to include js at the bottom of the file

Reply:
The headscripts.js in Sakai is intended to be included at the beginning.  Hence the name "headscripts" - it is tiny and kept that way for really basic utility code that is often needed in the body of the code.